.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
```

Comment:
Hi @sazukeR , well done!

Please change the `ubuntu-version` from `ubuntu-18.04` to `ubuntu-22.04`